amundsen_application/static/js/components/common/ResourceListItem/SchemaInfo/styles.scss

```diff
@@ -0,0 +1,5 @@
+@import 'variables';
+
+.underline{
+  border-bottom: solid #9191A8;
```

Comment:
is there a variable for that color?

Reply:
I have bot checked yet, I was trying to put here working code. Do you think I should put this style in _layout.scss file as underline may be used at other places for standardization? or should I create a new stylesheet like above?

Reply:
Maybe you want to bubble it up to the ResourceListItem level, on its styles.scss file.

You will probably be able to find a variable with that color.

Reply:
For some reason I can't find the actual hex color value in Figma, let's double check with design about what color this is. Then either find it or enter it in `_variables-default.scss`. I wonder if it's supposed to be the same color as $stroke-focus.

Reply:
Just to resolve this one, the color is $gray40.